pkg/build/api/validation/validation_test.go

```diff
@@ -1269,7 +1269,7 @@ func TestValidateCommonSpec(t *testing.T) {
 				Output: buildapi.BuildOutput{
 					To: &kapi.ObjectReference{
 						Kind: "DockerImage",
-						Name: "some/long/value/with/no/meaning",
+						Name: "index.docker.io/some/long/value/with/no/meaning",
```

Comment:
it's changing because this is now a valid image spec... only by pinning it to dockerhub can we trigger the invalid error?